lively-lions/MUD/items/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
```

Comment:
Great with inline comments, but if they were written as a docstring I think it would be more useful.